sunpy/image/slit.py

```diff
@@ -0,0 +1,171 @@
+import numpy as np
+
+import astropy.units as u
+
+
+def slit(mcube_in, range_a, range_b, N_slits=0, shift_x=+1, shift_y=-1,):
+    """
+    Returns an array with intensity along the slit on the y axis and time
+    along x.
+
+    Parameters
+    ----------
+    mcube_in : `sunpy.map.MapCube`
+        A mapcube of the images you want to perform the slit analysis on.
+        Usually with x and y as space, and z as time.
+        The x and y axes must be equal for all instances within the mapcube.
+    range_a : `astropy.units.Quantity`
+        A list of two `astropy.unit.Quantity` objects representing x1 and x2,
+        start and end of slit in x.
+    range_b : `astropy.units.Quantity`
+        A list of two `astropy.unit.Quantity` objects representing x2 and y2,
+        start and end of slit in y.
+    shift_x : `int`
+        Possible values, `-1`, `0`, `+1`
+        The displacement from the origin on the slit in x. These extra slits
+        calcuate a mean along each row to make the positioning of the original
+        slit less sensetive. Both `x` and `y` cannot be `0`. Default = `+1`
+    shift_y : `int`
+        Possible values, `-1`, `0`, `+1`
+        The displacement from the origin in `y`. Default = `-1`
+    N_slits : `int`
+        Number of deviations from central slit, e.g. to use 5 slits `N = 2`.
+        Default = 0
+
+    Returns
+    -------
+    im_array : `numpy.ndarray`
+        A numpy array with intenisty in y and time in x.
+    slit : `numpy.ndarray`
+        A numpy array representing the indicies of the central slit.
+
+    """
+
+    # check the attributes of the coordinates
```

Comment:
This comment is wrong/misleading, this block is ensuring the range input is in pixels.